docs/docs/howto/vault.md

```diff
@@ -1,224 +1,175 @@
-They will have to be filled in manually.
-If you don't know the correct values use `ffffffffffffffffffffffffffffffff` and `000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000` respectively and observe the real values in the error messages in the following steps.
-This should only be done in a secure environment.
+If you added SANs in the previous step, these will also be included in the certificate.
+In that case, you can configure your DNS service to resolve the additional SAN to the load balancer IP.
 
-:::
+Alternatively, configure the system from which you access Vault to resolve `vault` to the load balancer IP.
+On Linux, you can modify `/etc/hosts` for this.
 
-</TabItem>
-</Tabs>
+### Initialize Vault
 
-:::note[Runtime class and Initializer]
-
-The deployment YAML shipped for this demo is already configured to be used with Contrast.
-A [runtime class](../architecture/components/runtime.md) `contrast-cc` was added to the pods to signal they should be run as Confidential Containers.
-During the generation process, the Contrast [Initializer](../architecture/components/initializer.md) will be added as an init container to these workloads.
-It will attest the pod to the Coordinator and fetch the workload certificates and the workload secret.
-
-:::
-
-### Deploy the Coordinator
-
-Deploy the Coordinator resource first by applying its resource definition:
+A Vault instance needs to be initialized once before it can be used.
+Initialization is a sensitive operation that transfers key material, and should thus be executed on a secure system, by a trusted operator and over a secure channel.
+For this demo, the machine hosting your Contrast workspace is sufficient.
+Initialize the Vault with the `bao` CLI from within your workspace, using the Coordinator CA as a root of trust:
 
 ```sh
-kubectl apply -f deployment/coordinator.yml
+export VAULT_ADDR=https://${YOUR_VAULT_DOMAIN}:8200
+export VAULT_CACERT=./coordinator-root-ca.pem
+bao operator init
 ```
 
-### Set the manifest
+Upon successful initialization, this prints a root token and some recovery key shares.
+These are highly sensitive secrets and need to be guarded carefully!
+
+### Configure Vault
 
-Configure the Coordinator with a manifest.
-It might take up to a few minutes for the load balancer to be created and the Coordinator being available.
+Vault can be configured to use Contrast certificates for authorization.
+The following commands enable [certificate authentication](https://developer.hashicorp.com/vault/docs/auth/cert) and assign a policy `contrast` to workloads with Contrast certificates.
 
 ```sh
-coordinator=$(kubectl get svc coordinator -o=jsonpath='{.status.loadBalancer.ingress[0].ip}')
-echo "The user API of your Contrast Coordinator is available at $coordinator:1313"
-contrast set -c "${coordinator}:1313" deployment/
+export VAULT_TOKEN=${YOUR_ROOT_TOKEN}
```

Comment:
```suggestion
export VAULT_TOKEN="<your_root_token>"
```

`YOUR_ROOT_TOKEN`  doesn't appear to exist, and the notation suggests it's an existing var